src/chains/ethereum/ethereum/src/miner/miner.ts

```diff
@@ -231,9 +232,9 @@ export default class Miner extends Emittery.Typed<
         if (event.opcode.name === "SSTORE") {
           const key = TraceData.from(
             event.stack[event.stack.length - 1].toArrayLike(Buffer)
-          );
-          const hashedKey = Data.from(keccak(key.toBuffer())).toString();
-          storageKeys.set(hashedKey, key.toBuffer());
+          ).toBuffer();
+          const hashedKey = keccak(key);
+          storageKeys.set(hashedKey.toString(), { key, hashedKey });
```

Comment:
Trading some memory for CPU. Previously the consumer of this data would convert the Map's keys (`hashedKey.toString()`) back into Buffers before saving them to the database. Now it just reads the Buffers stored in the values of the Map.